spec/models/building_spec.rb

```diff
@@ -1,11 +1,16 @@
 require 'rails_helper'
 
 RSpec.describe Building, type: :model do
-  it "has name, location_latitude, location_longitude" do
-    building = FactoryBot.create :building
+  building = FactoryBot.create :building
+  openingtime = FactoryBot.create(:openingtime, timeable: building)
```

Comment:
should be fixture. Again, unrelated to locations?

Reply:
not sure what "fixtrue" means, but I agree we shouldn't have changed that on this branch.

Reply:
We changed it to a fixture now.
It doesn't really belong to this branch maybe, but we will leave it here.